pkg/rancher-components/src/components/BadgeState/BadgeState.vue

```diff
@@ -74,27 +74,30 @@ export default defineComponent({
   .badge-state {
     align-items: center;
     display: inline-flex;
-    padding: 2px 10px;
+    padding: 2px 8px;
     border: 1px solid transparent;
     border-radius: 20px;
+    height: 20px;
 
     &.bg-info {
-      border-color: var(--info);
+      color: var(--on-info-banner);
```

Comment:
i got lost on this. in components we 

- introduce new colours with no fallback on an old colour
- when vars have been renamed however we do fallback on old an colour (which now does not exist)

should we either always fallback, or it's not needed in both places?

Reply:
We will need to define a fallback for new colors where one wasn't defined before. FYI @edenhernandez-suse